vmdb/app/models/ems_refresh/save_inventory_configuration.rb

```diff
@@ -0,0 +1,34 @@
+# Calling order for EmsForeman
+#
+# - ems
+#   - configuration_profiles (foreman HostGroup)
+#     - operating_system_flavor (links)
+#     - customization_script_ptable (link)
+#     - customization_script_medium (link)
+#   - configured_systems (foreman Host)
+#     - operating_system_flavor (links)
+#     - customization_scripts (links)
+## ? use guid?
+module EmsRefresh
+  module SaveInventoryConfiguration
+    def save_configuration_manager_inventory(manager, hashes, target = nil)
+      return if hashes.nil?
+      save_child_inventory(manager, hashes, [:configuration_profiles, :configured_systems], target)
+      manager.save
+    end
+
+    def save_configuration_profiles_inventory(manager, hashes, target)
+      hash_contains_all_values = target.nil? || manager == target
```

Comment:
oh i see...yeah, it's confusing because this stuff is introducing the targeting concept, even though targeting doesn't exist for anything but vmware.

yeah, delete_missing_records reads much nicer, or maybe `process_all_deletes` or something.